app/src/main/res/values-de/strings.xml

```diff
@@ -296,6 +296,12 @@ Andernfalls kannst du eine alternative Tastatur im App Store herunterladen. Beli
     <string name="quest_powerPolesMaterial_wood">"Holz"</string>
     <string name="quest_powerPolesMaterial_metal">"Stahl"</string>
     <string name="quest_powerPolesMaterial_concrete">"Beton"</string>
+
+    <string name="quest_bbq_fuel_title">Womit wird dieser Grill betrieben?</string>
```

Comment:
translations are only imported from translation site, any data here will be overwritten by such import

if you can translate it: it is really welcome to add it to translation site after PR is merged and new version is being prepared

Reply:
Thank you, i will skip this step next time